File: task1/main.kt
fun main() {
    print("Введите строку: ")
    val text = readln().lowercase() // Читаем ввод и приводим к нижнему регистру

    val vowels = "aeiouyаеёиоуыэюя" // Все гласные
    val consonants = "bcdfghjklmnpqrstvwxzбвгджзйклмнпрстфхцчшщ" // Все согласные

    var vowelCount = 0
    var consonantCount = 0

    text.forEach { char -> // Для каждого символа в строке
        when {
            char in vowels -> vowelCount++
            char in consonants -> consonantCount++
        }
    }

    // Выводим результат сравнения
    println(
        when {
            vowelCount > consonantCount -> "да"
            vowelCount < consonantCount -> "нет"
            else -> "одинаково"
        }
    )
}

File: task1/main.php
<?php

echo "Введите строку: ";
$text = strtolower(trim(fgets(STDIN))); // Читаем ввод и приводим к нижнему регистру

$vowels = "aeiouyаеёиоуыэюя"; // Все гласные
$consonants = "bcdfghjklmnpqrstvwxzбвгджзйклмнпрстфхцчш"; // Все согласные

$vowelCount = 0;
$consonantCount = 0;

foreach (str_split($text) as $char) { // Для каждого символа в строке
    if (strpos($vowels, $char) !== false) {
        $vowelCount++;
    } elseif (strpos($consonants, $char) !== false) {
        $consonantCount++;
    }
}

// Выводим результат сравнения
if ($vowelCount > $consonantCount) {
    echo "да";
} elseif ($vowelCount < $consonantCount) {
    echo "нет";
} else {
    echo "одинаково";
}
?>
